repository-ydb-v2/src/main/java/tech/ydb/yoj/repository/ydb/YdbRepositoryTransaction.java

```diff
@@ -575,4 +581,42 @@ public String toString() {
             }
         });
     }
+
+    /**
+     * @return YDB SDK {@link YdbTransaction} wrapping this {@code YdbRepositoryTransaction}
+     */
+    @ExperimentalApi(issue = "https://github.com/ydb-platform/yoj-project/issues/80")
+    public YdbTransaction toSdkTransaction() {
```

Comment:
As an option: we could add `RepositoryTransaction.toNativeTransacton` to avoid explicit casting.

Anyway, changes are approved.